_test/test_gen_sqw_workflow/test_gen_sqw_accumulate_sqw_herbert.m

```diff
@@ -50,9 +50,9 @@
                 test_name = mfilename('class');
             end
             combine_algorithm = 'mpi_code'; % this is what should be tested
-            if is_jenkins && ispc
-                combine_algorithm = 'mex_code'; % disable mpi combine on Jenkins. It is extreamly slow.
-            end
+            %if is_jenkins && ispc
```

Comment:
should this section just be deleted?

Reply:
theoretically, job should not depend on framework, and I have verified that it does not. The filebased MPI is indeed very slow for pixel combining so it probably  better to enable this block again.